src/common/transformations/include/transformations/rt_info/disable_fp16_compression.hpp

```diff
@@ -32,4 +38,20 @@ class TRANSFORMATIONS_API DisableFP16Compression : public RuntimeAttribute {
     }
 };
 
+/**
+ * @ingroup ie_runtime_attr_api
+ * @brief PostponedFP16Compression class represents runtime info attribute that marks operation
+ * as prohibitted to convert to FP16 as part of Compressed Only format.
+ */
+class TRANSFORMATIONS_API PostponedFP16Compression : public RuntimeAttribute {
+public:
+    OPENVINO_RTTI("postponed_fp16_compression", "0");
+
+    PostponedFP16Compression() = default;
+
+    bool is_copyable() const override {
+        return false;
+    }
+};
```

Comment:
Please don't introduce another `RuntimeAttribute`-derived class. It is too heavy for such a simple task. One of the disadvantages is that this attribute will appear in serialized xml and we don't need it. Another one is a new specialization of Any will be generated producing unnecessary binary code bloating. The whole story with this `RuntimeAttribute`-derived class is too bulky to further extend it. Please just use a simple text tag without any classes -- keep set/get methods below.

Reply:
agreed and remvoed